### PR TITLE
172 create api controller for listinginquiries

### PR DESCRIPTION
**Pull request by JohanSaisa**

### CODE REVIEW

File: src/GT.API/Controllers/InquiryController.cs [hunk 1]
```diff
@@ -1,12 +1,126 @@
...
+			if (dto is null)
+			{
+				return NotFound();
+			}
+
+			var result = JsonConvert.SerializeObject(dto);
+
+			return Ok(result);
+		}
+
```
[the remark this answers is not in the document]
Comment (JohanSaisa) [reply src/GT.API/Controllers/InquiryController.cs:59]: Added in new commit.

File: src/GT.API/Controllers/InquiryController.cs [hunk 2]
```diff
@@ -1,12 +1,126 @@
...
+			{
+				await _inquiryService.UpdateAsync(dto, id);
+				return Ok();
+			}
+			catch (Exception)
+			{
+				return StatusCode(500);
+			}
+		}
+
```
[the remark this answers is not in the document]
Comment (JohanSaisa) [reply src/GT.API/Controllers/InquiryController.cs:104]: Added in new commit.

File: src/GT.API/Controllers/InquiryController.cs [hunk 3]
```diff
@@ -1,12 +1,126 @@
...
+			if (dtos is null)
+			{
+				return NotFound();
+			}
+
+			var result = JsonConvert.SerializeObject(dtos);
+
+			return Ok(result);
+		}
+
```
[the remark this answers is not in the document]
Comment (JohanSaisa) [reply src/GT.API/Controllers/InquiryController.cs:38]: Added in new commit.

**Review by OscarThorburn — APPROVED** [review]

**Comment by JohanSaisa** [comment]
[US5001: Developer - Inquiry API](https://trello.com/c/DXQS359t/11-us5001-developer-inquiry-api)

**Pull request merged**